docs/guides/ingesters-scaling-up-and-down.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Ingesters scaling up and down"
+linkTitle: "Ingesters scaling up and down"
+weight: 10
+slug: ingesters-scaling-up-and-down
+---
+
+This guide explains how to scale up and down ingesters.
+
+_If you're looking how to run ingesters rolling updates, please refer to the [dedicated guide](./ingesters-rolling-updates.md)._
+
+## Scaling up
+
+Adding more ingesters to a Cortex cluster is considered a safe operation. When a new ingester startups, it will register to the [hash ring](../architecture.md#the-hash-ring) and the distributors will reshard received series accordingly.
```

Comment:
```suggestion
Adding more ingesters to a Cortex cluster is considered a safe operation. When a new ingester starts, it will register to the [hash ring](../architecture.md#the-hash-ring) and the distributors will reshard received series accordingly.
```